### DISCUSSION TITLE
Integration of a Trigger in DataFactoryTools for the Publish Branch

### DISCUSSION THREAD

**Discussion by EkueAF** [Azure-Player/azure.datafactory.tools#414] [Q&A]
Dear Kamil,

I hope this message finds you well. 

I am reaching out to discuss the possibility of integrating a trigger in your tool, DataFactoryTools, specifically for the publish branch. 

Our aim is to deploy the collaboration branch as suggested in your documentation, but we would like this to occur when we publish the collaboration branch to the published version

Could you please advise if this is feasible?

Looking forward to your guidance on this matter.

Best regards,
Ekué

**Comment by NowinskiK** [comment]
This is the trigger you must configure for your code repository (Azure DevOps or GitHub).
Either in Classic UI as well as YAML files in Azure DevOps you can define which branch is being monitored for changes. 
Then, react on this trigger (DevOps, not ADF trigger) and run build+release pipelines to deploy new code.
